src/Symfony/Component/Console/Command/Command.php

```diff
@@ -461,6 +496,21 @@ public function addOption(string $name, string|array $shortcut = null, int $mode
         return $this;
     }
 
+    /**
+     * Set values for input completion.
+     *
+     * @throws InvalidArgumentException If the option is not defined
+     *
+     * @return $this
+     */
+    public function setOptionValues(string $name, \Closure|iterable|null $values): static
```

Comment:
Actually, let's remove both setters. Using only the constructor argument is enough and reduces the API surface appropriately to me.